expo/package.json

```diff
@@ -36,5 +33,9 @@
     "eslint": "8.10.0",
     "eslint-config-prettier": "6.0.0",
     "jest": "26.6.3"
+  },
+  "peerDependencies": {
+    "@datadog/mobile-react-native": ">=1.0.0",
+    "expo": "46.x"
```

Comment:
so how we are going to do the versioning? we are going to release `45.0.1` from the dedicated branch as well to support Expo 45?

Reply:
yes exactly, as this version would not work with expo 45